src/sdk/strategy.ts

```diff
@@ -177,36 +168,32 @@ export class DeployedStrategy {
     const encryptedTreasureMap = EncryptedTreasureMap.fromBytes(
       policy.encryptedTreasureMap
     );
+    const aliceVerifyingKey = PublicKey.fromBytes(policy.aliceVerifyingKey);
     const newPolicy = {
       id,
       label: policy.label,
       policyKey,
       encryptedTreasureMap,
-      aliceVerifyingKey: new Uint8Array(policy.aliceVerifyingKey),
+      aliceVerifyingKey: aliceVerifyingKey.toBytes(),
       size: policy.size,
       startTimestamp: policy.startTimestamp,
       endTimestamp: policy.endTimestamp,
       txHash: policy.txHash,
     };
-    const aliceSecretKey = SecretKey.fromBytes(aliceSecretKeyBytes);
     const bobSecretKey = SecretKey.fromBytes(bobSecretKeyBytes);
     const label = newPolicy.label;
     const cohort = Cohort.fromObj(cohortConfig);
-    const porterUri = cohort.configuration.porterUri;
-    const configuration = { porterUri };
-    const alice = Alice.fromSecretKey(configuration, aliceSecretKey, provider);
     const encrypter = new Enrico(
       newPolicy.policyKey,
-      alice.verifyingKey,
+      aliceVerifyingKey,
```

Comment:
Hmm, it looks like Alice's  (secret?) keys are unintentionally being passed to Enrico here.

Reply:
`const aliceVerifyingKey = PublicKey.fromBytes(policy.aliceVerifyingKey);`